### ISSUE TITLE
hp.plotter crashes

### ISSUE THREAD

**Issue by Chenshenfei** [paulvangentcom/heartrate_analysis_python#96]
Why when I use hp.plotter to plot in pycharm, the plot box flashes back, and so does hp.plot_poincare.

**Comment by DingDingZhao** [comment]
Hi, I have a very similar issue, and then I switched to Jupyter and the hp.plotter seems working but there is a warning like "UserWarning: Matplotlib is currently using module://ipykernel.pylab.backend_inline, which is a non-GUI backend, so cannot show the figure." but the figure still shows

**Comment by Chenshenfei** [comment]
Yes, my problems with Jupyter are the same as yours. I ran it on Pycharm and needed to add a line "plt.show()" to the end of the program.

**Comment by paulvangentcom** [comment]
I will install PyCharm and have a look over the weekend, strange.
